src/main/java/de/konfidas/ttc/messages/systemlogs/DisableSecureElementSystemLogMessage.java

```diff
@@ -0,0 +1,84 @@
+package de.konfidas.ttc.messages.systemlogs;
+
+import de.konfidas.ttc.exceptions.BadFormatForLogMessageException;
+import de.konfidas.ttc.messages.SystemLogMessage;
+import de.konfidas.ttc.messages.logtime.GeneralizedLogTime;
+import de.konfidas.ttc.messages.logtime.LogTime;
+import de.konfidas.ttc.messages.logtime.UnixLogTime;
+import de.konfidas.ttc.messages.logtime.UtcLogTime;
+import org.bouncycastle.asn1.*;
+
+import java.io.IOException;
+import java.text.ParseException;
+import java.util.Collections;
+import java.util.List;
+import java.util.ListIterator;
+import java.util.NoSuchElementException;
+
+
+/**
+ * Diese Klasse repräsentiert eine disableSecureElementSystemLog Message. Dabei werden in der Methode
+ * parseSystemOperationDataContent die folgenden Elemente aus systemOperationData geparst
+ * <pre>
+ * ╔═══════════════════════╤══════╤═══════════════════════════════════════════════════════════════╤════════════╗
+ * ║ Data field            │ Tag  │ Data Type                                                     │ Mandatory? ║
+ * ╠═══════════════════════╪══════╪═══════════════════════════════════════════════════════════════╪════════════╣
+ * ║ timeOfDeactivation    │ 0x81 │ Time                                                          │ m          ║
+ * ╚═══════════════════════╧══════╧════════════════════════════════════════════════════════════════════════════╝
+ * </pre>
+ */
```

Comment:
This breakes javadoc again.

Reply:
Works on my machine :-)
How do you generate the javadoc?